### PR TITLE
UtxoId changes

### PR DESCRIPTION
**Pull request by rakita**
changes for UtxoId: https://github.com/FuelLabs/fuel-tx/pull/53

### CODE REVIEW

**Comment by adlerjohn** [comment]
Is this good to go?

**Comment by rakita** [comment]
It is still not good to go, i want to check some more small things, and want to Brandon to look at it so that he is familiar with it, it is more impactful than fuel-vm.

File: Cargo.lock [hunk 1]
```diff
@@ -1510,9 +1510,9 @@ dependencies = [
  "env_logger",
  "fuel-asm",
  "fuel-storage",
- "fuel-tx",
+ "fuel-tx 0.1.0",
```
Comment (Voxelot) [inline Cargo.lock:1513]: it seems like we're getting mismatched versions of fuel-tx somehow

Comment (rakita) [reply Cargo.lock:1513]: Probably some library that uses `fuel-tx` or `fuel-vm` is not updated, will check.

Comment (rakita) [reply Cargo.lock:1513]: fueltx_01

It is used by example project `simple-wasm` that imports `fuels-abigen-macro`->`forc`->`fuel-gql-client`. 
To to do the update, we need to merge `fuel-gql-client` and update version to `forc`/`fuel-abigen-macro`

Comment (adlerjohn) [reply Cargo.lock:1513]: Given how messy this is, and how the example project is more of an integration thing, should it simply be removed from the workspace?

Comment (rakita) [reply Cargo.lock:1513]: Seems resonable to be removed or moved to somewhere else if still needed (not sure where).

**Review by adlerjohn — APPROVED** [review]
Changes to the lockfile can be clobbered in #140.

File: Cargo.lock (hunk 1 above)
Comment (adlerjohn) [reply Cargo.lock:1513]: #140

**Pull request merged**